__tests__/fixtures/list/dev-deps-prod/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "left-pad": "^1.1.3"
```

Comment:
Instead of relying on "real" dependencies, you can just construct local packages which may simplify and speed up things.